tensorboard/program.py

```diff
@@ -466,7 +516,7 @@ def server_bind(self):
         # since that's expected if IPv4 isn't supported at all (IPv6-only).
         if hasattr(errno, 'EAFNOSUPPORT') and e.errno != errno.EAFNOSUPPORT:
           logger.warn('Failed to dual-bind to IPv4 wildcard: %s', str(e))
-    super(WerkzeugServer, self).server_bind()
+    serving.ThreadedWSGIServer.server_bind(self)
```

Comment:
Comment for why this isn't a call to super()?

Reply:
No longer needed.